src/RileyMaze/toolbox.js

```diff
@@ -67,21 +71,6 @@ var RMZToolbox = GObject.registerClass(class RMZToolbox extends Toolbox {
         this._unitTopic.reset();
     }
 
-    _onGameStateChanged(self, senderName, [key, value]) {
```

Comment:
I don't see any callers of this method being removed elsewhere in this commit, so I suspect it should either be reinstated or the callers should be removed.

Reply:
I hadn't added any callers because the game state wasn't in place for the game yet. Since I'm locking the topic buttons instead of hiding them and the quest isn't using 'topic' in any keys I didn't end up using the method for anything.

Reply:
Oh, okay.